packages/generator-react-server/generators/app/templates/routes.js

```diff
@@ -1,10 +1,11 @@
+const path = require('path');
 
 module.exports = {
 	routes: {
 		HelloWorld: {
 			path: ['/'],
 			method: 'get',
-			page: './pages/hello-world',
+			page: path.join('.', 'pages', 'hello-world'),
```

Comment:
So, the error was `Module not found: Error: Cannot resolve module '..pageshello-world' in C:\mydev\reactserver__clientTemp`.  What sticks out to me is that there are _no_ slashes in the error message _at all_, which makes me think that the `/` is being interpreted as an escape character.  I'm hoping that `path.join` has the logic to do the right thing.